app/assets/stylesheets/widgets/discussion-post.scss

```diff
@@ -2,6 +2,16 @@
 @import 'mixins';
 
 .widget-discussion-post {
+    display:flex;
+
+    > .post-avatar {
```

Comment:
Would you mind using `&>` for these please, just to keep it consistent in the codebase (I personally find it more readable that way, although I know it's not "better")